test/handler.test.js

```diff
@@ -472,6 +498,36 @@ describe('strong-error-handler', function() {
           done);
     });
 
+    it('sanitizes response to prevent XSS', function(done) {
```

Comment:
Please make this test name more descriptive - e.g. `HTML-escapes response fields to prevent XSS in production mode`.